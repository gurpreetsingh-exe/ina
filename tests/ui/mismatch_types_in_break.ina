// COMPILE_FAIL

fn main() {
    loop {
        break 20;
        break 20.;
    };
}
