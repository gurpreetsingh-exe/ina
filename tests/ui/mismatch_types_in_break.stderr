error: mismatch types
   ╭─[tests/ui/mismatch_types_in_break.ina:6:15]
   │
 6 │         break 20.;
   │               ─┬─
   │                ╰── expected `integer`, found `float`
   ∙
───╯
